test/csuite/wt4105_large_doc_small_upd/main.c

```diff
@@ -149,7 +149,8 @@ main(int argc, char *argv[])
             modify_entry.offset = offset;
             modify_entry.size = modify_entry.data.size;
             /* FIXME-WT-6113: extend timeout to pass the test */
-            (void)alarm(15);
+            if (!testutil_is_flag_set("TESTUTIL_MSAN"))
+                (void)alarm(15);
```

Comment:
Why do we need to avoid this for the MSan variant?

Reply:
This alarm will trigger due to msan significatnly slowing down the performance when optimisation is off. The alarm is there to catch any performance regressions but I don't think we should be concerned about that when testing with msan.

Reply:
I'll add a comment.

Reply:
Oh I see - that makes sense now, thanks for explaining that. Yes, I agree a comment is a good idea.